### PR TITLE
Updates for AlertmanagerConfig and PrometheusRule

### PR DESCRIPTION
**Pull request by StevenBarre**
- How to create long term silences
- How to customize alert receivers
- Update the list of alerts to match https://github.com/bcgov-c/devops-fulfillment-pipeline/pull/185
- Update the frequency to match https://github.com/bcgov-c/devops-fulfillment-pipeline/pull/185

### CODE REVIEW

File: src/docs/app-monitoring/user-defined-monitoring.md [hunk 1]
```diff
@@ -213,6 +206,62 @@ Please have this network policy added to ensure that proper metrics are scraped
...
+  - name: business-hours
+    timeIntervals:
+    - times: # Times are in UTC; 08-16 PST / 09-17 PDT
+      - startTime: "16:00"
+        endTime: "23:59"
+      weekdays:
+      - monday:friday
+```
+
+If you want to use an email receiver, keep it the same an only update the `name` and `to` fields, as well as the `Product Name Here` in the `html`. The rest ensures that the email is sent correctly and with nice formatting.
```
Comment (tbaker1313) [inline src/docs/app-monitoring/user-defined-monitoring.md:257]:
```suggestion
If you want to use an email receiver, keep it the same and only update the `name` and `to` fields, as well as the `Product Name Here` in the `html`. The rest ensures that the email is sent correctly and with nice formatting.
```

**Review by wmhutchison — APPROVED** [review]
👍

**Review by tbaker1313 — APPROVED** [review]

File: src/docs/platform-automation/alertmanager.md [hunk 2]
```diff
@@ -76,29 +96,36 @@ Alerts are triggered only if the condition persists for 1 hour, minimizing unnec
 
 In the **Developer** perspective, select **Observe** -> **<project_name>** -> **Alerts**. In this perspective, alerts, silences, and alerting rules are all managed from the **Alerts** page. The results shown in the **Alerts** page are specific to the selected project.
 
-## Objects
+## Silencing Alerts
```
Comment (caggles) [inline src/docs/platform-automation/alertmanager.md:99]:
```suggestion
## Silencing alerts
```

Pilar says best practice is to capitalize only the first word in a title (or proper nouns, etc)

File: src/docs/platform-automation/alertmanager.md [hunk 3]
```diff
@@ -17,33 +17,55 @@ content_owner: Olena Mitovska
...
-* **[View in web console](#view-in-web-console)**
-* **[Objects](#objects)**
-* **[Timeline](#timeline)**
-* **[Related pages](#related-pages)**
+
+- [Alert contacts](#alert-contacts)
+- [Frequency](#frequency)
+- [Alerts](#alerts)
+- [View in web console](#view-in-web-console)
+- [Silencing Alerts](#silencing-alerts)
```
Comment (caggles) [inline src/docs/platform-automation/alertmanager.md:33]:
```suggestion
- [Silencing alerts](#silencing-alerts)
```

**Review by caggles — CHANGES_REQUESTED** [review]
looks good, just two small updates suggested, and then I think we could merge this!

**Review by caggles — APPROVED** [review]

**Pull request merged**